.travis.yml

```diff
@@ -88,10 +88,12 @@ install:
 script:
   - |
     if [ -z "$STACK_YAML" ]; then
-       cabal new-test
+      cabal new-test
     else
       stack build --test --no-terminal
     fi
+  - curl -o .hlint-relude.yaml https://raw.githubusercontent.com/kowainik/relude/master/.hlint.yaml
+  - curl -sSL https://raw.github.com/ndmitchell/neil/master/misc/travis.sh | sh -s -- hlint -h .hlint-relude.yaml .
```

Comment:
This is really suprising that we don't have any hints in `summoner` and our code is perfect :thinking: :thinking: :thinking: 
But I've checked it locally, looks like we really don't have any hints

Reply:
That's the outcome for me as well - I copied the `.hlint.yaml` from `relude` and it reports "No hints".